doc/source/cookbook/custom_data_types.rst

```diff
@@ -65,9 +65,8 @@ Submitted by Kevin R. Thornton.
 
 
     if __name__ == '__main__':
-        x = np.array(np.random.rand(100))
-        x=np.reshape(x,(50,2))
-        x.dtype=[('x',np.float),('y',np.float)]
+        x = np.array(np.random.rand(100)).reshape(50,2)
```

Comment:
```suggestion
        x = np.random.rand(100).reshape(50,2)
```

Reply:
Right, that's even nicer. I'll update the commit with that.